distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -35,7 +35,7 @@ After adding permissions, a system restart is required for them to take effect.
 . permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
 . permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read, write";
 
-Trailing slashes after <DIRECTORY_PATH> have no effect on the permissions granted. For example, adding a permission for "/test/path" and "/test/path/" are equivalent. The recursive forms "/test/path${/}-", and "/test/path/${/}-" are also equivalent.
+Trailing slashes after <DIRECTORY_PATH> have no effect on the permissions granted. For example, adding a permission for "{/}test{/}path" and "{/}test{/}path{/}" are equivalent. The recursive forms "{/}test{/}path${/}-", and "{/}test{/}path${/}-" are also equivalent.
```

Comment:
Should all of the slashes have `${/}` instead of just `{/}`?

Reply:
good catch. fixed